AutomatedOnlineFoodOrdering/Controllers/Store_Folder/StoreController.cs

```diff
@@ -29,12 +30,14 @@ public PartialViewResult ViewCategory()
             }
         }
 
+
         public ActionResult StoreIndex(int? page, int? category)
         {
+
             using (DBModels dbModel = new DBModels())
             {
                 var pageNumber = page ?? 1;
-                var pageSize = 10;
+                var pageSize = 9;
```

Comment:
Why are you making size 9?

Reply:
because i want it to be three per row and i have not uploaded much food to the site that is why.

Reply:
so i will still change it